src/views/Pools/index.tsx

```diff
@@ -17,6 +17,7 @@ import PoolTabButtons from './components/PoolTabButtons'
 import BountyCard from './components/BountyCard'
 
 const Pools: React.FC = () => {
+  useFetchCakeVault()
```

Comment:
Populating the vault Redux store. Opted to do it within the Pools view rather than app.tsx but easily changed if you'd prefer that?

Reply:
I think this is what we should do now moving forward with the exception of data that is used many places throughout the site.